src/components/molecules/result-card.tsx

```diff
@@ -1,132 +1,72 @@
-import React from 'react';
-import {StyleSheet, Text, View} from 'react-native';
-import {useNavigation} from '@react-navigation/native';
-import {useTranslation} from 'react-i18next';
-import * as WebBrowser from 'expo-web-browser';
+import React, {FC} from 'react';
```

Comment:
This result card is a **brand new card** that's supposed to be used in all places where the screen is just a confirmation. The current implementation is a fragile one, which only covers the success page of the callback feature.

The previous "result-card" was renamed to "symptom-checker-result-card" and it is untocuhed.